### PR TITLE
LGA-2415 confirm diversity data encryption

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

Add celery task to decrypt diversity data with current private key and log whether that was a success
Add command to schedule celery task to decrypt diversity data

## Any other changes that would benefit highlighting?

Add rabbitmq as an additional docker compose service
Add celery workers as an additional docker compose service

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_backend/apps/cla_butler/models.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class DiversityDataCheck(TimeStampedModel):
+    personal_details = models.OneToOneField(PersonalDetails)
+    detail = models.TextField(null=True, blank=True)
+    action = models.CharField(max_length=20)
+    status = models.CharField(max_length=10)
+
+    @classmethod
+    def get_unprocessed_personal_data_qs(cls):
+        return PersonalDetails.objects.prefetch_related('diversitydatacheck').filter(
```
Comment (said-moj) [inline cla_backend/apps/cla_butler/models.py:14]: This need to filter for check records only

**Review by Pheonnexx — APPROVED** [review]
LGTM 🚀

**Review by psweeting1 — COMMENTED** [review]
Tiny comment, welcome to ignore it. If so, let me know and I'll approve.

File: cla_backend/apps/cla_butler/models.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from model_utils.models import TimeStampedModel
+from django.db import models
+from django.db.models import Q
```
[the remark this answers is not in the document]
Comment (said-moj) [reply cla_backend/apps/cla_butler/models.py:3]: Q is the default module name and also commonly referred to as Q

**Pull request merged**